src/controllers/namespace.go

```diff
@@ -23,64 +24,65 @@ type namespaceController struct {
 	logger *zap.Logger
 }
 
-func NewNSController(client *kubernetes.Clientset, logger *zap.Logger) NameSpaceController {
+func New(client *kubernetes.Clientset, logger *zap.Logger) NamespaceController {
 	return &namespaceController{
 		logger: logger,
 		client: client,
 		ctx:    context.Background(),
```

Comment:
We should probably not have the `context.Background()` here in the struct and instead pass the `gin context` to the functions separately

Reply:
Thanks!